src/app/helptext/system/tunable.ts

```diff
@@ -28,7 +28,7 @@ export const helptextSystemTunable = {
       'Enter a value to use for the <a\
  href="https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/boot-introduction.html#boot-loader-commands"\
```

Comment:
@Mrt134 Need advice on what to replace these links with.

Reply:
I'm looking at the related area in the SCALE UI (22.02-RC.2) and noticing the section has been retitled "Sysctl" and no longer has a drop-down for switching between loader, sysctl, or rc.conf options. 

![image](https://user-images.githubusercontent.com/17933320/148087481-303aa106-4e4c-4afd-9d89-5fc2d8d62448.png)

If I'm reading this correctly, then the entire tooltip can be removed except for the loader link: "Enter a value to use for the <a href="https://man7.org/linux/man-pages/man8/sysctl.8.html" target="_blank">sysctl</a>."

Reply:
Makes sense, fixed.

Reply:
@Mrt134 Bump.